src/core/meshcop/commissioner.cpp

```diff
@@ -779,6 +779,7 @@ void Commissioner::HandleJoinerFinalize(Coap::Header &aHeader, Message &aMessage
     (void)aMessageInfo;
     StateTlv::State state = StateTlv::kAccept;
     ProvisioningUrlTlv provisioningUrl;
+    uint8_t buf[OPENTHREAD_CONFIG_MESSAGE_BUFFER_SIZE];
```

Comment:
Thanks for the suggestion, I have added the CERT_LOG macro definition to wrap the dumping code.